lib/controller.js

```diff
@@ -7,7 +7,10 @@ var sprintf = require('sprintf-js').sprintf;
 var cp = require('child_process');
 var async = require('async');
 var updates = require('./update-fetch');
+var debug = require('debug')('controller');
+var Menu = require('terminal-menu');
 var controller = {};
+controller.TESSEL_AUTH_KEY = Tessel.TESSEL_AUTH_PATH + '/id_rsa';
```

Comment:
> I'm always partially thinking about how we can make this CLI eventually usable for other types of devices and to do that we would need to separate the device key location from the command parsing.

@johnnyman727  That is a good idea. In that case we should have a second and third VM to be discovered. That way we can hold this in mind.

By writing this, I got awash by TODOs like
- replace all "Tessel" by DEVICETYPENAME
- TESSEL_AUTH_PATH should become '.'+DEVICETYPENAME+AUTH_PATH
- ...++

What type of devices ? Tessel based or also Ra. PI, Intel an others ?
Or do you mean to be able to fork and easy re-engineering this CLI for e.g. Ra. PI for running OpenWRT on it ?